commands/dev.js

```diff
@@ -60,6 +60,9 @@ exports.run = async function(
         // $FlowFixMe
         testRuntime ? testRuntime.run() : Promise.resolve(),
       ]);
+      if (!open) {
```

Comment:
Thanks for the feature addition. I wonder if we should just show this message all the time, whether or not it opened.

Reply:
I also think showing message all time will be better in case accidentally one closes tab then they can always look on the terminal, please let me know if i should update PR